api/v1alpha1/edgedeployment_types.go

```diff
@@ -33,6 +33,7 @@ type EdgeDeploymentSpec struct {
 	Data            *DataConfiguration             `json:"data,omitempty"`
 	ImageRegistries *ImageRegistriesConfiguration  `json:"imageRegistries,omitempty"`
 	Metrics         *ContainerMetricsConfiguration `json:"metrics,omitempty"`
+	LogCollection   string                         `json:"log_collection,omitempty"`
```

Comment:
Please add comment/docs what that string is supposed to be and what is its relation to configs in the EdgeDevice.